projects/1.18/assets/chileancraft/chileancraft/lang/zh_cn.json

```
@@ -0,0 +1,871 @@
+{
+  "advancements.alabordaje.descr": "获得一柄水手的佩剑",
+  "advancements.alabordaje.title": "上船吧，孩子！",
+  "advancements.estamos_bien_los_33.descr": "遇见一名迷路的矿工",
+  "advancements.estamos_bien_los_33.title": "他大概33岁",
+  "advancements.fundandopomaire.descr": "放置一台陶工车床",
```

```suggestion
  "advancements.fundandopomaire.descr": "放置一台陶轮",
```
https://es.wikipedia.org/wiki/Torno_de_alfarero